janitor/spark/backend.py

```diff
@@ -63,7 +63,10 @@ def register_dataframe_accessor(name):
     ----
     Modified based on pandas.core.accessor.
     """
-    from pyspark.sql import DataFrame
+    try:
+        from pyspark.sql import DataFrame
+    except ImportError:
+        import_message("pyspark", "conda install pyspark")
```

Comment:
This follows the usage of `import_message`, but it would be cool in the future if `import_message` knew to use pip/conda for non-specific packages (`pyspark` is a good example)

Reply:
@hectormz let's raise that as a new issue.

That said, I think the use of import_message here might be missing its first argument. Suggested change:

```suggestion
        import_message("spark", "pyspark", "conda install pyspark")
```

Reply:
Sounds good, I can raise that issue. 👍